src/translations/en.json

```diff
@@ -48,5 +48,13 @@
     "playAiContent2":"to learn the ropes and test your mettle.",
     "gameName": "Game Name",
     "join":"Join"
+  },
+  "game":{
+    "dialogs":{
```

Comment:
Let's change this to `"game.menus.spectatorListMenu"` so that we can group it with the `GameMenu` content in a future update and separate it from the dialogs